webapp/store/user/hooks.js

```diff
@@ -9,6 +10,10 @@ import { useSurveyInfo } from '@webapp/store/survey'
 import * as UserState from './state'
 
 export const useUser = () => useSelector(UserState.getUser)
+export const useIsSystemAdminUser = () => {
+  const user = useSelector(UserState.getUser)
+  return !User.isSystemAdmin(user)
```

Comment:
1. call the hook useUserIsSystemAdmin
2. if the hook returns true if the user is systemAdmin, why do you return !User.isSystemAdmin(user)
3. can this just be:

```typescript
export const useUserIsSystemAdmin = () => User.isSystemAdmin(useUser())
```

Reply:
ahahah I was trying this case.., and just leave the ! ... thanks!